src/components/ClusterHealthManager/ClusterHealthManager.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react';
+import useSessionStore from 'state/session';
+import { useClusterNodeStatus } from 'src/swr/jwt';
+import { FSStats, LoadAverage, NodeStatusInfo } from 'types/cratedb';
+
+export type ClusterHealthManagerProps = {
+  clusterId: string;
+};
+
+export const STATS_PERIOD = 15 * 60 * 1000;
+
+function ClusterHealthManager({ clusterId }: ClusterHealthManagerProps) {
+  const { clusterHealth, setClusterHealth } = useSessionStore();
+  const { data: nodes } = useClusterNodeStatus(clusterId);
+  const [prevNodeData, setPrevNodesData] = useState<NodeStatusInfo[] | []>();
+
+  const getFs = (nodes: NodeStatusInfo[]): { [key: string]: FSStats } => {
```

Comment:
I think that

```suggestion
  const getFs = (nodes: NodeStatusInfo[]): Record<string, FSStats> => {
```

is more readable